content/post/050-ipfs-camp-recap.md

```diff
@@ -0,0 +1,87 @@
+---
+date: 2019-07-05
+url: 2019-07-05-ipfs-camp-recap
+tags: IPFS, IPFS Camp
+title: 2019 IPFS Camp Recap
+author: David Dias
+---
+
+Greetings InterPlanetary Community!
+
+The InterPlanetary Space Training Camp is over and now that we are all back to Earth, I have the joy of sharing the news and learnings with you.
+
+If you aren't familiar with the [InterPlanetary Space Training Camp](https://camp.ipfs.io/) (IPFS Camp for short), it was a 3 day Hacker retreat for the Builders of the DWeb. Attendees travelled through a [space portal](https://twitter.com/ChrisPacia/status/1144291810874920967) to get to Space Training Station in another Galaxy (Star coordinates yet undisclosed). Throughout the Camp, attendees had the chance to participate in:
+
+#### [**👩🏽‍🏫 Core & Elective Courses**](https://github.com/ipfs/camp#-core--elective-courses)
+> Specialized and focused trainings on building for the DWeb.
+
+![](https://ipfs.io/ipfs/QmUiucP3oRVnqvuM6hnrF9D5H6tRSeVhyWk6ggkhf4ebEH)
```

Comment:
@momack2 following on your suggestion/request. What about having a photo collage of 3 highlights from each format?

Reply:
added!